cluster/gce/upgrade-aliases.sh

```diff
@@ -124,9 +129,9 @@ function add-k8s-subnet-secondary-ranges() {
 function delete-k8s-node-routes() {
   local -a routes
   local -r batch=200
-  routes=( $(gcloud compute routes list \
-    --project=${PROJECT} --filter='description=k8s-node-route' \
-    --format='value(name)') )
+  mapfile -t routes < <(gcloud compute routes list \
+     --project="${PROJECT}" --filter='description=k8s-node-route' \
+     --format='value(name)')
```

Comment:
why change: 
```
In ./cluster/gce/upgrade-aliases.sh line 127:
  routes=( $(gcloud compute routes list \
           ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```
Ref: https://github.com/koalaman/shellcheck/wiki/SC2207